data/35e/wizards_of_the_coast/supplement/dragon_magic/dm_classes.lst

```diff
@@ -14,5 +14,9 @@ CLASS:Dragonfire Adept	STARTSKILLPTS:4	CSKILL:Appraise|Bluff|Climb|Concentration
 2	ABILITY:Dragonfire Adept Class Feature|AUTOMATIC|Dragonfire Adept ~ Breath Effect
 2	ABILITY:Dragonfire Adept Class Feature|AUTOMATIC|Dragonfire Adept ~ Scales
 4	ABILITY:Dragonfire Adept Class Feature|AUTOMATIC|Dragonfire Adept ~ Dragonkin
-6	ABILITY:Dragonfire Adept Class Feature|AUTOMATIC|Dragonfire Adept ~ Damage Reduction
+6	ABILITY:Dragonfire Adept Class Feature|AUTOMATIC|Dragonfire Adept ~ Damage Reduction	DR:2/magic
+8	BONUS:VAR|ScalesArmorBonus|1
```

Comment:
The DR should be placed in the ability and then bonused. The BONUS:VAR should be turned into a formula and placed into the correct ability.